src/css/sections/hero.css

```diff
@@ -0,0 +1,177 @@
+.hero-wrapper {
```

Comment:
hero-wrapper on desktop should be 1152px but now 1024

Reply:
use negative margin or set `.hero .container { paddong: 0 }`

Reply:
also please add for .header

.header { margin-bottom: 8px)

min-width: 768px 
    .header { margin-bottom: 32px)
    
min-width: 1152px 
    .header { margin-bottom: 16px)